src/components/Dashboard.js

```diff
@@ -7,8 +9,10 @@ class Dashboard extends Component {
   }
   render() {
     return (
-      <div>
+      <div className="container">
```

Comment:
There is no `.container` class in your CSS. Did you want to use the `Container` component from `reactstrap`?

Reply:
its `bootstrap` class. we can use `Container` component as well which works same as `.container` class

Reply:
I've never used `reactstrap` before. It's really weird that you include the whole bootstrap file plus you use components. I think you should choose between the two (either bootstrap classes or components everywhere). It was kind of misleading 😄